### ISSUE TITLE
Using one repo as IBM SoftLayer

### ISSUE THREAD

**Issue by gyliu513** [IBM/cluster-api-provider-ibmvpccloud#6]
We already have a repo at https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud for IBM SoftLayer, maybe we can consider if this one can be merged togethers into same repo as https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud

@wentao-zh ^^

**Comment by gyliu513** [comment]
My major concern is the open source community may have some concern as we already have https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud in kubernetes sigs, so the community may have question why not using one repo to handle all IBM Cloud.

**Comment by spzala** [comment]
I agree with @gyliu513 that we should keep everything under the community repo. My understanding is this repo is for POC (if not, we should keep that way) and we will be moving the code to the community repo soon we can.

**Comment by wentao-zh** [comment]
@gyliu513 @spzala  It makes sense. We can put IBM VPC gen2 infrastructure, IBM classic infrastructure, PowerVS and maybe other types of iaas into  cluster-api-provider-ibmcloud.
And also we can build our kubeadmcontrolplane to bootstrap IKS rather than only vanilla k8s.

**Comment by spzala** [comment]
@wentao-zh awesome, thanks!

**Comment by gyliu513** [comment]
@wentao-zh cluster api version for ibmcloud is very low https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/master/go.mod#L16 , this needs to be updated. It may request some SL knowledge as well.
